src/pages/interface/TransactionDetails.vue

```diff
@@ -43,7 +43,7 @@
       :src="checkmark"
       alt="green checkmark"
     >
-    {{ $("InterfaceTransactionDetails.status.value.success") }}
+    {{ $t("InterfaceTransactionDetails.status.value.success") }}
```

Comment:
already on main

Reply:
@cacampbell That probably came from main, I merged main into this branch recently.